### PR TITLE
revert multizone etcd setup

### PR DESCRIPTION
**Pull request by linki**
make etcd secure again

### CODE REVIEW

**Comment by hjacobs** [comment]
:+1:

File: cluster/etcd-cluster.yaml [hunk 1]
```diff
@@ -19,7 +17,6 @@ SenzaComponents:
       source: registry.opensource.zalan.do/acid/etcd-cluster:3.0.15-p14
```
Comment (CyberDem0n) [inline cluster/etcd-cluster.yaml]: registry.opensource.zalan.do/acid/etcd-cluster:3.0.17-p15

Comment (hjacobs) [inline cluster/etcd-cluster.yaml]: changed

**Comment by hjacobs** [comment]
:+1:

**Comment by lmineiro** [comment]
👍

**Comment by linki** [comment]
thanks @CyberDem0n for checking

In fact, for teapot, I updated the eu-central stack. Then deleted the eu-west stack. Then rolled each node one by one.

**Pull request merged**